extensions/vtk/src/utils/setMPRLayout.js

```diff
@@ -1,11 +1,21 @@
 import setLayoutAndViewportData from './setLayoutAndViewportData.js';
 
-export default function setMPRLayout(displaySet) {
+export default function setMPRLayout(
+  displaySet,
+  viewportPropsArray,
+  numRows = 1,
+  numColumns = 3
+) {
   return new Promise((resolve, reject) => {
     const viewports = [];
-    const numRows = 1;
-    const numColumns = 3;
     const numViewports = numRows * numColumns;
+
+    if (viewportPropsArray && viewportPropsArray.length !== numViewports) {
+      reject(
```

Comment:
Can we reject with `new Error('viewportProps is supplied but its length is not equal to numViewports')` instead of just a string?